src/core/thread/mle_tlvs.hpp

```diff
@@ -1139,6 +1140,12 @@ typedef SimpleTlvInfo<Tlv::kChannel, ChannelTlvValue> ChannelTlv;
  */
 typedef SimpleTlvInfo<Tlv::kCslChannel, ChannelTlvValue> CslChannelTlv;
 
+/**
+ * Defines Wake-up Channel TLV constants and types.
+ *
+ */
+typedef SimpleTlvInfo<Tlv::kWakeupChannel, ChannelTlvValue> WakeupChannelTlv;
```

Comment:
If I am not mistaken, the newly defined `Mle:: WakeupChannelTlv ` is not used within this PR? 

Should we add it in a sperate PR (when used)?

Reply:
Right, only used in MeshCoP space.